.semaphore/release/release.yml

```diff
@@ -45,8 +46,6 @@ blocks:
         - sudo systemctl stop docker
         - sudo umount /var/lib/docker && sudo killall qemu-nbd || true
         - sudo systemctl start docker
-        # Free up space on the build machine.
-        - sudo rm -rf ~/.kiex ~/.phpbrew ~/.rbenv ~/.nvm ~/.kerl ~/.sbt ~/.npm /usr/lib/jvm /opt/firefox* /opt/apache-maven* /opt/scala /usr/local/golang
```

Comment:
Not for this job, because for this build we're just building felix and a few other components and then uploading them. We're not doing anything close to a full build.

The Semaphore VMs we use for this build job start with 62 GB free space (before checkout); removing these files frees up 2.1 GB, but building the OpenStack packages only requires about 2.0 GB (including checkout) in ~semaphore, plus about 2.1 GB for docker containers.